src/VolleyManagement.Backend/VolleyManagement.Services/Authentication/VolleyUserStore.cs

```diff
@@ -302,12 +302,12 @@ private void Map(User to, UserModel from)
             to.Email = from.Email;
             to.PersonName = from.PersonName;
             to.PhoneNumber = from.PhoneNumber;
-            to.LoginProviders = from.Logins.ConvertAll(
+            to.LoginProviders = ((List<LoginProviderModel>)from.Logins).ConvertAll(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to 'List<LoginProviderModel>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1905)